packages/rocketchat-livechat/client/views/app/livechatCurrentChats.html

```diff
@@ -52,6 +53,13 @@
 							<td>{{startedAt}}</td>
 							<td>{{lastMessage}}</td>
 							<td>{{status}}</td>
+							{{#requiresPermission 'remove-closed-livechat-rooms'}}
+								{{#if isClosed}}
+									<td><a href="#remove" class="remove-livechat-room"><i class="icon-trash"></i></a></td>
+								{{else}}
+									<th>&nbsp;</th>	
+								{{/if}}						
+							{{/requiresPermission}}									
```

Comment:
please add an `{{else}}` block with an empty `<td>` so it does not break the table if the user does not have the required permission